_posts/en/2020-05-20-Team-Topologies-Whitelisting-app.md

```diff
@@ -3,59 +3,61 @@ layout: post
 
 For this project, a manager acted as the product owner for each of the teams, the Value Stream Aligned Team, the DTS Platform Team, and the second SE Platform Team. Please note this does not need to be the case. Actually, ideally it should not be the case. The product owner (the one responsible for prioritizing the work of the team) should be someone from business (albeit with technical knowledge or understanding). One model often referenced is the so-called, and poorly named, “Spotify model”, which is only used here as a demonstration of an alternate model whereby product owners and managers needn't be the same person. While this is generally the model that was used, it is abstracting away complexities that make it difficult for this model to work well in a government context. For example, the director at the top who is responsible for the management of the product is not the director for all of the teams in the box, though he is responsible for the platform, which makes reporting, hierarchies, and performance management a potential issue.
 
-![Compare curves]({{ site.baseurl }}/assets/images/TTBlog-Figure3.PNG)
+![Spotify Model]({{ site.baseurl }}/assets/images/TTBlog-Figure3.PNG)
 _Figure 3_
 
-In the above model, the squads are managed by Product Owners (POs) while the chapters would be managed by managers. The person located in the top right could be out director, or we could say our Product Manager. Once we merge the above model with the governmentized Team Topologies model, we could produce a more desirable model that looks something as follows
+In the above model, the squads are managed by Product Owners (POs) while the chapters would be managed by managers. The person located in the top left could be the director. In this case, they would play the role of the Product Manager. Once we merge the above model with the governmentized Team Topologies model, we could produce a more desirable model that looks something as follows
 
-![Compare curves]({{ site.baseurl }}/assets/images/TTBlog-Figure4.PNG)
+![Final Government Team Topologies Model]({{ site.baseurl }}/assets/images/TTBlog-Figure4.PNG)
 _Figure 4_
 
-To make this easier to view we have removed details from the initial model, though it still applies. For example, the method of communication, though long term the interactions between teams should strive to be XaaS. That is, the interactions between teams are automated as a service that is provided to other teams. Further, there is the possibility to add enabling teams to the vertical teams identified here.
+To make this easier to view we have removed details from the initial model, though it still applies. For example, the method of communication, though long term the interactions between teams should strive to be through automated means. That is, the interactions between teams are automated as a service that is provided to other teams. Further, there is the possibility to add enabling teams to the vertical teams identified here.
```

Comment:
wondering if the second sentence would be better as "For example, regarding the method of communication, the ultimate goal would be for interactions between teams to be through automated means"

Reply:
Updated the phrasing to what you recommended here